samples/bluetooth/mesh/dfu/distributor/src/main.c

```diff
@@ -12,12 +12,17 @@
 #include <zephyr/bluetooth/bluetooth.h>
 #include <zephyr/bluetooth/mesh.h>
 #include <zephyr/bluetooth/mesh/shell.h>
+#include <bluetooth/mesh/vnd/le_pair_resp.h>
 #include <bluetooth/mesh/dk_prov.h>
 #include <dk_buttons_and_leds.h>
 #include "smp_bt.h"
 #include "dfu_dist.h"
 #include "dfu_target.h"
 
+#if !defined(CONFIG_MCUMGR_TRANSPORT_BT_AUTHEN) || !defined(CONFIG_BT_MESH_LE_PAIR_RESP)
+#pragma warning "Authentication for SMP over BLE is disabled"
```

Comment:
Doesn't just `#warning` work?

Reply:
Moved to `printk` as warning makes CI to fail